storm-core/test/clj/backtype/storm/security/auth/ThriftClient_test.clj

```diff
@@ -20,26 +20,26 @@
   (:import [org.apache.thrift.transport TTransportException])
 )
 
+(def TIMEOUT (Integer. (* 3 1000)))
```

Comment:
> Are we going from 30 seconds to 3000 miliseconds here? Shouldn't it be 30000?

30 seconds is an extremely long timeout for a thrift response.  Most of the time, 30ms is enough.  If the thrift server does not reply within 3 full seconds, I am done waiting for it.


> How about adding a comment describing why the choice of 3000 in this particular test?

Yes, I can add a shorter comment.